compiler/src/dotty/tools/dotc/reporting/messages.scala

```diff
@@ -3399,3 +3399,7 @@ class GivenSearchPriorityWarning(
        |$migrationHints"""
 
   def explain(using Context) = ""
+final class EnumMayNotBeValueClasses(sym: Symbol)(using Context) extends SyntaxMsg(EnumMayNotBeValueClassesID):
```

Comment:
Blank line please :-p
```suggestion
  def explain(using Context) = ""

final class EnumMayNotBeValueClasses(sym: Symbol)(using Context) extends SyntaxMsg(EnumMayNotBeValueClassesID):
```

Reply:
Ah, it was a merge conflict with #22189 and the blank line was dropped when resolving the conflict 😅